tests/testhelpers.h

```diff
@@ -40,3 +40,29 @@ void verifyMatrix (GpMatrix *matrix, REAL e1, REAL e2, REAL e3, REAL e4, REAL e5
 #define createWchar(c) L ##c
 #define freeWchar(c)
 #endif
+
+#define assertEqualInt(actual, expected) assertEqualIntImpl (actual, expected, __FILE__, __LINE__)
+void assertEqualIntImpl (INT actual, INT expected, const char *file, INT line)
+{
+    if (actual != expected)
+    {
+        printf ("Assertion failed on line %d in %s\n", line, file);
+        printf ("Expected: %d\n", actual);
+        printf ("Actual:   %d\n", expected);
```

Comment:
- You switched actual and expected here.
- We should write to stderr instead of stdout like `assert()`

Reply:
Thanks for noticing. I've fixed this.